### PR TITLE
Creating project folders

### PR DESCRIPTION
**Pull request by aima98**
- Re-creating Awesome books app following the rules:
      1. Using ES6 syntax.
      2. Dividiing your code into modules.
      3. Creating a separate file for each module in the modules directory.
      4. Importing modules into the index.js file.
      5. Keeping all funcionalities of the app without errors.
      6. Refactoring your functions to arrow functions.
      7. Using let and const in the correct way.
      
- Making sure that there are o linter errors.
- Making sure that correct GitHub Flow are used.
- Making sure that the work is documented in a professional way.
- Following HTML, CSS and JavaScript best practices.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,139 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome_Books_ES6] <a name="about-project"></a>
+
+> This capstone project was built for Microverse [under these guidance objectives](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) > _note: you can only access this link if you're a Micronault (Microverse Student)_
+
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly remove this section of your readme file as it is referring to the HTML capstone project

File: modules/book.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    `;
+    list.appendChild(bookDiv);
+  }
+
+  static deleteBook(element) {
+    if (element.classList.contains('rmv-btn')) {
+      element.parentElement.remove();
+    }
+  }
+
```
Comment (julie-ify) [inline modules/book.js]: - [ ] This project is an ES6 refactor of the previous projects. That means the functionalities, by implication, are required to stay the same. When I delete a book it is only removed from the page, but not from the local storage. So when I refresh the page the removed books are added back to the page. Kindly fix that. User actions should be persisted in the storage

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @aima98,

Good job so far!

## Highlights!
- Book addition functionality

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md]: well resolved! ✔️

File: modules/book.js (hunk 2 above)
Comment (rloterh) [reply modules/book.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Oumaima,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- App works as intended!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly ensure that the _Getting Started_ section_ contains the appropriate instructions on how to _setup_ and _run_ this app.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**